backend/src/main/java/com/woowacourse/friendogly/pet/dto/request/SavePetRequest.java

```diff
@@ -9,9 +9,10 @@
 import jakarta.validation.constraints.PastOrPresent;
 import jakarta.validation.constraints.Positive;
 import jakarta.validation.constraints.Size;
-import java.time.LocalDate;
 import org.hibernate.validator.constraints.URL;
 
+import java.time.LocalDate;
+
 public record SavePetRequest(
         @NotNull(message = "memberId는 null을 입력할 수 없습니다.")
         @Positive(message = "memberId는 음수일 수 없습니다.")
```

Comment:
```suggestion
        @NotNull(message = "memberId는 null을 입력할 수 없습니다.")
        @Positive(message = "memberId는 양수만 입력할 수 있습니다.")
```
사소한 부분이지만 수정하면 좋을듯요!